specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/mfe.json

```diff
@@ -24624,6 +24705,78 @@
       },
       "additionalProperties": false
     },
+    "ServerlessInferenceEndpoint": {
+      "required": [
+        "inferenceUri"
+      ],
+      "type": "object",
+      "properties": {
+        "headers": {
+          "description": "Specifies any required headers to target this serverless endpoint.",
+          "type": "object",
+          "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

Same feedback here. Can this be schematized? Otherwise it's difficult and risky for clients to consume.

Reply:
Same situation here applies. The content here is model dependent. It is intended as a generic contract to the client to populate these required headers when sending inferencing requests.

Reply:
To elaborate, the possible "headers" is unbounded based on customer (specifically, the model provider), so there is no way for us to specify all possible values in the schema upfront. I hope this is a valid use case of additional properties.